config/initializers/yaml_autoloader.rb

```diff
@@ -7,8 +7,14 @@
 #
 # Note, this is used to autoload constants serialized as yaml from one process and loaded in another such as through
 # args in the MiqQueue. An alternative would be to eager load all of our autoload_paths in all processes.
-Psych::Visitors::ToRuby.prepend Module.new {
-  def resolve_class(klass_name)
-    (class_loader.class != Psych::ClassLoader::Restricted && klass_name && klass_name.safe_constantize) || super
-  end
-}
+#
+# This isn't needed with zeitwerk. To confirm, you can run this in rails console to see if it properly autoloads
+# from yaml:
+#    YAML.load("--- !ruby/object:MiqExpression\nexp:\ncontext_type:\ncol_details:\nruby:\n")
```

Comment:
Wonder if we can just make this a spec?  I know it's hard because you have to load something that can be loaded but won't even actually _be_ loaded.